stable/traefik/templates/NOTES.txt

```diff
@@ -11,3 +12,12 @@
          $ kubectl describe svc {{ template "fullname" . }} --namespace {{ .Release.Namespace }} | grep Ingress | awk '{print $3}'
 
 2. Configure DNS records corresponding to Kubernetes ingress resources to point to the load balancer IP/hostname found in step 1
+{{ end }}
+{{- if eq .Values.serviceType "NodePort" }}
+1. Traefik is listening on the following ports on the host machine:
+
+     http - {{ .Values.service.nodePorts.http }}
+     https - {{ .Values.service.nodePorts.https }}
```

Comment:
If you switch `serviceType` to `NodePort`, but don't specifically pick ports (i.e. ports are dynamically selected) then the above will render like so:

```
http - 
https - 
```

That's not too useful.